src/adapters/WstEthChainlinkAdapter.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity 0.8.21;
+
+import {IStEth} from "../interfaces/IStEth.sol";
+import {AggregatorV3Interface} from "../interfaces/AggregatorV3Interface.sol";
+
+import {ErrorsLib} from "../libraries/ErrorsLib.sol";
+
+/// @title WstEthChainlinkAdapter
+/// @author Morpho Labs
+/// @custom:contact security@morpho.org
+/// @notice wstETH/ETH exchange rate price feed.
+/// @dev This contract should only be used as price feed for `ChainlinkOracle`.
+contract WstEthChainlinkAdapter is AggregatorV3Interface {
+    uint8 public constant decimals = uint8(18);
```

Comment:
let's put a natspec, especially as I find this variable super misleading (`precision` would have been a much better name)

Reply:
We should add it to CL interface then no?

Reply:
nb: related to [this discussion](https://github.com/morpho-org/morpho-blue-oracles/pull/65#discussion_r1481803299)